### PR TITLE
Added prize section. Please review.

### PR DESCRIPTION
**Pull request by Olshia**

### CODE REVIEW

**Comment by sahil9001** [comment]
Resolve conflicts

File: src/sections/section9/Section9.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+import './Section9.css';
+
+const Section9 = ()=> {
+
+    return (
+        <div className="codeutsava_section9">
+            <div className="codeutsava_section9-title">
+                <div className="codeutsava_section9-title1">Prizes</div>
+                <div className="codeutsava_section9-title2">Win excitinng prizes worth upto 8 lakhs</div>
```
Comment (sahil9001) [inline src/sections/section9/Section9.js:14]: rename excitinng to exciting

**Review by sahil9001 — CHANGES_REQUESTED** [review]

**Comment by sahil9001** [comment]
Pull the latest changes

**Comment by sahil9001** [comment]
Umm I think you merged the wrong branch, please pull the `dev` branch instead of `main`.

**Comment by sahil9001** [comment]
@Olshia Merge `dev` branch changes

File: src/sections/section9/Section9.js (hunk 1 above)
Comment (Olshia) [reply src/sections/section9/Section9.js]: Done

**Pull request merged**